### PR TITLE
[DOCS] Allow attribute substitution in titleabbrevs for Asciidoctor migration

### PR DESCRIPTION
**Pull request by jrodewig**
By default, Asciidoctor does not expand attributes included in `<titleabbrev>` tags. This Adds the `[subs="attributes"]` to `<titleabbrev>` tags containing attributes in preparation for the Asciidoctor migration

Relates to /elastic/docs#827 and #41128.

### CODE REVIEW

**Comment by nik9000** [comment]
I believe you can also enable [subs="attributes"] on the +++ snippets.
Either way is fine with me.

**Comment by jrodewig** [comment]
Thanks @nik9000. I went with the `[subs="attributes"]` option in 8ca7685 as that seemed less destructive.

File: docs/reference/data-frames/apis/delete-transform.asciidoc [hunk 1]
```diff
@@ -2,8 +2,10 @@
 [testenv="basic"]
 [[delete-data-frame-transform]]
 === Delete {dataframe-transforms} API
+
+[subs="attributes"]
 ++++
-<titleabbrev>Delete {dataframe-transforms}</titleabbrev>
+<titleabbrev>Delete data frame transforms</titleabbrev>
```
Comment (nik9000) [inline docs/reference/data-frames/apis/delete-transform.asciidoc]: I think you can revert this line now that you have `subs=attributes`.

File: docs/reference/data-frames/apis/get-transform-stats.asciidoc [hunk 2]
```diff
@@ -2,8 +2,10 @@
 [testenv="basic"]
 [[get-data-frame-transform-stats]]
 === Get {dataframe-transform} statistics API
+
+[subs="attributes"]
 ++++
-<titleabbrev>Get {dataframe-transform} statistics</titleabbrev>
+<titleabbrev>Get data frame transform statistics</titleabbrev>
```
Comment (nik9000) [inline docs/reference/data-frames/apis/get-transform-stats.asciidoc]: Same for all of the others actually.

**Review by nik9000 — CHANGES_REQUESTED** [review]

File: docs/reference/data-frames/apis/delete-transform.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/data-frames/apis/delete-transform.asciidoc]: :doh: Sorry about that @nik9000. For some reason, I thought I changed both the titles and titleabbrevs. Looks like I only did the titles though.

Should be fixed up with 71460d7.

**Review by nik9000 — APPROVED** [review]
LGTM

**Pull request merged**